server/test/test_scanpy_engine.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+
+from server.app.scanpy_engine.scanpy_engine import ScanpyEngine
+
+
+class UtilTest(unittest.TestCase):
+    def setUp(self):
+        self.data = ScanpyEngine("example-dataset/", schema="data_schema.json")
+
+    def test_init(self):
+        assert self.data.cell_count == 2638
```

Comment:
is there a reason this is used instead of `self.assertEqual`?  i think assertEqual will produce nicer messages when they fail (i.e., it'll print out the two values).

Reply:
no reason ¯\_(ツ)_/¯, I'll make that change